test/runtests.jl

```diff
@@ -3,18 +3,31 @@ using DistributionsAD
 using Combinatorics
 using Distributions
 using FiniteDifferences
-using ForwardDiff
 using PDMats
-using ReverseDiff
-using Tracker
-using Zygote
+
+# Figure out which AD backend to test
+const AD = get(ENV, "AD", "All")
+if AD == "All"
+    @eval using ForwardDiff
+    @eval using ReverseDiff
+    @eval using Tracker
+    @eval using Zygote
+elseif AD == "Zygote"
+    @eval using Zygote
+elseif AD == "ReverseDiff"
+    @eval using ReverseDiff
+elseif AD == "ForwardDiff_Tracker"
+    @eval using ForwardDiff
+    @eval using Tracker
+else
+    error("Unknown AD backend: $AD")
+end
```

Comment:
Maybe just
```suggestion
if AD == "All" || AD == "ForwardDiff"
    @eval using ForwardDiff
end
if AD == "All" || AD == "Zygote"
    @eval using Zygote
end
if AD == "All" || AD == "ReverseDiff"
    @eval using ReverseDiff
end
if AD == "All" || AD == "Tracker"
    @eval using Tracker
end
```

Reply:
BTW we have to add an additional Github action if we split ForwardDiff and Tracker.